bin/civihr-install

```diff
@@ -713,10 +713,13 @@ export PATH="$PATH:$BINDIR"
 
 civihr_parse_options "$@"
 
+# Checking for Git dependencies.
+command -v git >/dev/null 2>&1 || ( echo "Please download git to continue." && exit 99 )
```

Comment:
Did you test this? It doesn't seem to be working. It prints the "Please download git to continue" message but the installation continues:

![captura de tela de 2017-02-16 07-32-18](https://cloud.githubusercontent.com/assets/388373/23015686/e74c0778-f41a-11e6-8432-50ee6283b32a.png)

I believe the problem is that the `exit` command is inside a subshell, so it will exit that subshell and not the one where the script is running